armi/reactor/tests/test_components.py

```diff
@@ -921,6 +1032,13 @@ def test_thermallyExpands(self):
 
 
 class TestHexagon(TestShapedComponent):
+    """Test hexagon shaped component.
+
+    .. test:: Hexagon shaped component
+        :id: T_ARMI_COMP_SHAPES1
+        :tests: R_ARMI_COMP_SHAPES
+    """
+
```

Comment:
Again, delete this.  Or move it down to one-or-more of the tests in this class.

We can't have class-level test crumbs. Not your fault, you didn't know.